i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-windows-snmp.md

```diff
@@ -5,100 +5,169 @@ title: Windows SNMP
 
-## Contenu du pack de supervision
+* **OS-Windows-SNMP-custom**
 
-### Objets supervisés
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
 
-Tous les systèmes d'exploitation Microsoft Windows sont supportés:
+<Tabs groupId="sync">
+<TabItem value="OS-Windows-SNMP-custom" label="OS-Windows-SNMP-custom">
 
-  - Windows serveur (2003, 2012, 2016)
-  - Windows client (7, 8, 10)
-  - ...
+| Alias  | Modèle de service             | Description                                                                                                                                                                  |
+|:-------|:------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Cpu    | OS-Windows-Cpu-SNMP-custom    | Contrôle du taux d'utilisation du CPU de la machine. Ce contrôle pourra remonter la moyenne du taux d'utilisation des CPU ainsi que le taux par CPU pour les CPU multi-coeur |
+| Memory | OS-Windows-Memory-SNMP-custom | Contrôle du taux d'utilisation de la mémoire vive                                                                                                                            |
+| Swap   | OS-Windows-Swap-SNMP-custom   | Contrôle du taux d'utilisation de la mémoire virtuelle                                                                                                                       |
 
-### Règles de découvertes
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **OS-Windows-SNMP-custom** est utilisé.
 
-<Tabs groupId="sync">
-<TabItem value="Host" label="Host">
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
 
-| Nom de la règle                 | Description                                                                         |
-| ------------------------------- | ----------------------------------------------------------------------------------- |
-| App-Protocol-SNMP-HostDiscovery | Découvrez vos serveurs Windows en scannant les agents SNMP sur un sous-réseau donné |
+| Alias                | Modèle de service                           | Description                                                                                                                                                 | Découverte |
+|:---------------------|:--------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------:|
+| Disk-Generic-Id      | OS-Windows-Disk-Generic-Id-SNMP-custom      | Contrôle du taux d'espace libre disponible du disque. Pour chaque contrôle apparaîtra le nom du disque                                                      |            |
```

Comment:
l'ID, pas le nom?

Reply:
the ID is for the filter (not the displayed value)